package.xml

```diff
@@ -60,6 +60,7 @@
   <exec_depend>ros2_control</exec_depend>
   <exec_depend>rviz2</exec_depend>
   <exec_depend>xacro</exec_depend>
+  <exec_depend>kinova_gen3_7dof_robotiq_2f_85_moveit_config</exec_depend>
```

Comment:
should we add a config to moveit_resources?

Reply:
[I tried ](https://github.com/ros-planning/moveit_resources/pull/175)but got push back on this. Didn't want to set the precedence that if we want to modify the tutorial robot, it had to be added to `moveit_resources` which would balloon in size.

Reply:
I see :/